### PR TITLE
iOS Documentation: Add example adding multiple images

### PR DESCRIPTION
**Pull request by louwers**
Fun (I think) example that uses an MBTiles file added as a resource to the app, and shows images for different kinds of POIs.

The resulting `.mbtiles` file is available on our S3 bucket but the example also contains information how to generate it with the open data provided by the National Park Service.

Adapted from https://github.com/mapbox/ios-sdk-examples/blob/main/Examples/Swift/MultipleImagesExample.swift which just said "drag and drop this shapefile to Mapbox Studio".

### CODE REVIEW

File: platform/ios/app-swift/Sources/MultipleImagesExample.swift [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+
+        let imageLayer = MLNSymbolStyleLayer(identifier: "npc-poi-images", source: source)
+        imageLayer.sourceLayerIdentifier = "pois"
+        imageLayer.iconImageName = NSExpression(mglJSONObject: [
+            "match", ["get", "POITYPE"],
+            "Restroom", "restrooms",
+            "Trailhead", "trailhead",
+            "Viewpoint", "viewpoint",
+            "",
```
Comment (andresinaka) [inline platform/ios/app-swift/Sources/MultipleImagesExample.swift:55]: Does this work without doing the `["image", "restrooms"]`?

Related to: https://github.com/maplibre/maplibre-native/issues/331#issuecomment-2220468477

Documentation make is sound that we need to use the `["image", "image-name"]`. [Docs](https://maplibre.org/maplibre-style-spec/expressions/#image)

I'm confused! 🤔

**Review by andresinaka — APPROVED** [review]

File: platform/ios/app-swift/Sources/MultipleImagesExample.swift (hunk 1 above)
Comment (louwers) [reply platform/ios/app-swift/Sources/MultipleImagesExample.swift:55]: That would be true if there was a property `iconImage`, however in iOS you set `iconImageName` directly.

Comment (louwers) [reply platform/ios/app-swift/Sources/MultipleImagesExample.swift:55]: And yes it works...

Comment (louwers) [reply platform/ios/app-swift/Sources/MultipleImagesExample.swift:55]: Both work actually.

**Review by ianthetechie — COMMENTED** [review]
Just some minor wordsmithing nits; content looks good.

File: platform/ios/MapLibre.docc/MultipleImagesExample.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+The resulting `.mbtiles` file can be hosted with a tile server such as [Martin](https://martin.maplibre.org/) or embedded in the app bundle as a resource. Since the file is quite small in this case we will use that last option in this example. Martin comes with a [`mbtiles` binary](https://maplibre.org/martin/mbtiles-meta.html) that allows us to inspect what the MBTiles file contains from the command line.
+
+```
+mbtiles meta-all pois-nps.mbtiles
+```
+
+The important thing to note is that there is a `pois` layer whose features include a `POITYPE` attribute. While we could add icons for all kinds of `POITYPE` values included in the dataset, we will only add icons for restrooms, trailheads and viewpoints, and leave the rest as an exercise to the reader.
+
+The icons need to be extracted with a tool like [Inkscape](https://inkscape.org/) because the National Park Service includes all icons in a big vector file. This is outside the scope of this example.
```
[the remark this answers is not in the document]
Comment (louwers) [reply platform/ios/MapLibre.docc/MultipleImagesExample.md:31]: Explained here: https://github.com/maplibre/maplibre-native/blob/main/platform/ios/CONTRIBUTING.md#resources

**Review by ianthetechie — APPROVED** [review]

**Review by HarelM — APPROVED** [review]

**Pull request merged**